Resources/Prototypes/Entities/Clothing/Head/hats.yml

```
@@ -322,9 +322,9 @@
   description: A grand, stylish head of personnel's cap.
   components:
   - type: Sprite
-    sprite: Clothing/Head/Hats/hopcap.rsi
+    sprite: _Harmony/Clothing/Head/Hats/hopcap.rsi
```

```suggestion
    sprite: _Harmony/Clothing/Head/Hats/hopcap.rsi # Harmony change. Resprite
```

Upstream file changes require a comment.